en/02_Developer_Guides/15_Customising_the_Admin_Interface/07_ReactJS_Redux_and_GraphQL.md

```diff
@@ -1249,25 +1430,64 @@ const mutation = {
   templateName: CREATE,
   singularName: 'Note',
   pagination: false,
-  params: {},
+  params: {
+    input: 'CreateNoteInput!', // For graphql v3 use 'AppNoteCreateInputType!'
```

Comment:
Note: Yes, for graphql v3 you have to add a `input: 'AppNoteCreateInputType!'` parameter entry, because otherwise it wants to add it with `Input` as the parameter by default, and that will fail because the admin module overrides the field name to be in lowerCamelCase!